lib/classes/Service.js

```diff
@@ -164,14 +164,15 @@ class Service {
   }
 
   getEventInFunction(eventName, functionName) {
-    if (eventName in this.getFunction(functionName).events) {
-      return this.getFunction(functionName).events[eventName];
+    const event = this.getFunction(functionName).events.filter(event => Object.keys(event)[0] === eventName)[0];
```

Comment:
This is probably not the most elegant way to get that element but works. Let me know if you know any other solutions.